hal/nrf_nfct.h

```diff
@@ -267,6 +267,40 @@ typedef enum
     NRF_NFCT_SELRES_PROTOCOL_NFCDEP_T4AT = 3,  /**< NFC-DEP Protocol and Type 4A Tag platform). */
 } nrf_nfct_selres_protocol_t;
 
+/**
+ * @brief RFU7 (bit b8) configuration for the SEL_RES frame according to
```

Comment:
Do you have any use case when setting RFU bits is needed ?
I don't find any case when those bits need to have value different than 0.

Reply:
Some protocols use these bits, eg. Mifare Classic 1K needs to set RFU43 bits to 0b01. So for now I don't need the other bits than RFU43 really, but I figured that if one RFU is allowed to be set, then all should be allowed/possible.

Reply:
Ok then. I thought that there isn't any use case since I checked the Digital Spec.